troubleshoot/monitoring/node-moves-outages.md

```diff
@@ -9,33 +9,91 @@ products:
+
+Overall, having replicas and multiple availability zones helps minimize service interruption.
+
+### Data loss risk for non-HA deployments
+
+The system maintenance process always attempts to recover the vacated node's data from replicas or snapshots. However, if the deployment is not configured with HA, including replica shards, the maintenance process may not be able to recover the data from the vacated node.
+
+To minimize this risk, ensure your deployment follows the [high availability best practices](/deploy-manage/deploy/elastic-cloud/elastic-cloud-hosted-planning.md#ec-ha), which recommend:
+- Using at least two availability zones for production systems (three for mission-critical systems).
+- Configuring one or more replicas for each index (except for searchable snapshot indexes).
+
+As long as these recommendations are followed, system maintenance processes should not impact the availability of the data in the deployment.
+
+### Data loss risk without replica shards
+
+Configuring multiple availability zones helps your deployment remain available for indexing and search requests if one zone becomes unavailable. However, this alone does not guarantee data availability. To ensure that your data remains accessible, indices must be configured with [replica shards](/deploy-manage/distributed-architecture/clusters-nodes-shards.md).
+
+If an index has no replica shards and its primary shard is located on a node that must be vacated, data loss may occur if the system is unable to move the node gracefully during the maintenance activity.
+
+### Performance considerations during system maintenance
+
+The performance impact of system maintenance depends on how well the deployment is sized. Well-provisioned clusters with sufficient buffer capacity typically remain unaffected, while deployments already operating near their limits may experience slowdowns, or even intermittent request failures, during node vacating.
+
+High availability assumes not just redundancy in data and zones, but also the ability to absorb the loss or restart of a node without service disruption. As mentioned in [](/deploy-manage/deploy/elastic-cloud/elastic-cloud-hosted-planning.md#ec-ha):
+
+::::{admonition} Availability zones and performance
+Increasing the number of zones should not be used to add more resources. The concept of zones is meant for High Availability (2 zones) and Fault Tolerance (3 zones), but neither will work if the cluster relies on the resources from those zones to be operational.
+
+The recommendation is to **scale up the resources within a single zone until the cluster can take the full load (add some buffer to be prepared for a peak of requests)**, then scale out by adding additional zones depending on your requirements: 2 zones for High Availability, 3 zones for Fault Tolerance.
 ::::
```

Comment:
be careful about repeating info that's elsewhere - this concept is something we should probably use a snippet for 

you should also avoid bolding and brackets generally. "high availability" and "fault tolerance" also do not need Title Case.

"the recommendation is" is not an ideal sentence struture. Try "You should [blank]"

Reply:
the bolding is a relic of a previous time, and people keep adding it back into docs. We don't have capacity to do bulk style overhauls at all (we're focusing on accuracy right now, and those changes are low value/high effot) so I try to fix it when I see it. I see your point, but this is why I call out this kind of styling in the context of a review. just for future reference.